docker/php/Dockerfile

```diff
@@ -2,28 +2,45 @@
 # PHPDocker.io PHP 7.1 / FPM image #
 ####################################
 
-FROM phpdockerio/php71-cli
+FROM ubuntu:bionic-20210512
```

Comment:
Could we change this to: 
```FROM ubuntu:bionic```?

I don't know for how long ubuntu keeps its images based on the compilation date

Reply:
@davidbeig done! Good advice, didn't notice about this, but I'm sure it's important. I've tested it with ubuntu:bionic and it works OK.

After reading this documentation https://hub.docker.com/_/ubuntu?tab=description&page=1&ordering=last_updated it seems there are more possibilities, like ubuntu:18.04 but I think we're good with this one :)